src/components/ItemEditorPage/LeftPanel/Toasts/ItemAdded/ItemAddedToast.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useEffect, useState } from 'react'
+import { Button, Toast, ToastType } from 'decentraland-ui'
+import { Link, useHistory, useLocation } from 'react-router-dom'
+import { locations } from 'routing/locations'
+import { T, t } from 'decentraland-dapps/dist/modules/translation/utils'
+
+import ItemAddedLogo from '../../../../../icons/check.svg'
+
+import { Props } from './ItemAddedToast.types'
+import './ItemAddedToast.css'
+
+const ItemAddedToast: React.FC<Props> = props => {
+  const { collectionId } = props
+  const [shouldShowToast, setShouldShowToast] = useState(false)
+  const [item, setItem] = useState<string | undefined>()
+  const search = useLocation().search
+  const history = useHistory()
+
+  useEffect(() => {
+    const searchParams = new URLSearchParams(search)
+    const newItem = searchParams.get('newItem')
+    if (newItem) {
+      setShouldShowToast(true)
+      setItem(newItem)
+    }
+  }, [search])
+
+  function handleRemoveItemAddedToast() {
+    setShouldShowToast(false)
+    const searchParams = new URLSearchParams(search)
+    searchParams.delete('newItem')
+    history.replace({ search: searchParams.toString() })
```

Comment:
what do you think about moving this `replace` logic to an `ItemAddedToast.container.ts` and dispatching the replace like this: https://github.com/decentraland/builder/blob/master/src/components/LandDetailPage/LandDetailPage.container.ts#L25? would worth it? or maybe we should send the `newItem` in the props?

-- 
in a case worth it to create a container, we could use the previously created selector there